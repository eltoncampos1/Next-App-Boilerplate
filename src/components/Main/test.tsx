import { render, screen } from '@testing-library/react';
import Main from './index';

describe('<Main />', () => {
  it('Should Render heading', () => {
    const { container } = render(<Main />);

    expect(screen.getByRole('heading', { name: /react avan√ßado/i }));

    expect(container.firstChild).toMatchSnapshot();
  });
});
