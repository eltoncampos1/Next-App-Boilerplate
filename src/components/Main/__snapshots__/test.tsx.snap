// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should Render heading 1`] = `
<main>
  <h1>
    react avan√ßado
  </h1>
</main>
`;
